src/main/java/frc/robot/subsystems/drive/motors/CIMSteer.java

```diff
@@ -70,6 +87,6 @@ public void setPIDConstants(double kF, double kP, double kI, double kD){
   }
 
   public void setAngleOffset(Rotation2d angleOffset){
-    motor.setSelectedSensorPosition(motor.getSelectedSensorPosition() + angleToNative(angleOffset.minus(this.angleOffset).getDegrees()));
+    //motor.setSelectedSensorPosition(motor.getSelectedSensorPosition() + angleToNative(angleOffset.minus(this.angleOffset).getDegrees()));
```

Comment:
just make this function empty or get rid of it all together, nothing's really using it

Reply:
yup